lambda_function.py

```diff
@@ -0,0 +1,49 @@
+import json
+import os
+import logging
+from google.cloud import bigquery
+
+
+def lambda_handler(event):
+    messageBody = event["Records"]
+
+    for eachMessage in messageBody:
+
+        # extracting the body of the message
+        message = json.loads(eachMessage["Sns"]["Message"])[0]
```

Comment:
Check if `Sns` and `Message` are in the the row and throw and error before proceeding